packages/split-view/package.json

```diff
@@ -0,0 +1,56 @@
+{
+    "name": "@spectrum-web-components/split-view",
+    "publishConfig": {
+        "access": "public"
+    },
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/adobe/spectrum-web-components.git",
+        "directory": "packages/split-view"
+    },
+    "bugs": {
+        "url": "https://github.com/adobe/spectrum-web-components/issues"
+    },
+    "homepage": "https://adobe.github.io/spectrum-web-components/components/split-view",
+    "keywords": [
+        "spectrum css",
+        "web components",
+        "lit-element",
+        "lit-html"
+    ],
+    "version": "0.0.1",
+    "description": "",
+    "main": "src/index.js",
+    "module": "src/index.js",
+    "type": "module",
+    "exports": {
+        "./src/": "./src/",
+        "./custom-elements.json": "./custom-elements.json",
+        "./package.json": "./package.json",
+        "./sp-split-view": "./sp-split-view.js",
+        "./sp-split-view.js": "./sp-split-view.js"
+    },
+    "files": [
+        "custom-elements.json",
+        "*.d.ts",
+        "*.js",
+        "*.js.map",
+        "/src/"
+    ],
+    "sideEffects": [
+        "./sp-*.js",
+        "./sp-*.ts"
+    ],
+    "scripts": {
+        "test": "echo \"Error: run tests from mono-repo root.\" && exit 1"
+    },
+    "author": "",
+    "license": "Apache-2.0",
+    "devDependencies": {
+        "@spectrum-css/splitview": "^3.0.0-beta.5"
```

Comment:
There'll be a tiny race between this and #1169 as to the version that needs to be here. I'm not sure this PR can be updated without other updates in that PR, but the initial release of this pattern should use the `@3.0.0` release of the Spectrum CSS output. No change needed right now, but wanted to make sure I said this out loud.

Reply:
done with latest rebase, new: 
```
"devDependencies": {
        "@spectrum-css/splitview": "^3.0.0"
```